Implement new icon/color theme (#15769)
qbittorrent/qBittorrent: src/icons/README.md

# qBittorrent Icons
------------------------------------------

## Previous
Old icons were created by Bert Verhelst (<verhelstbert@gmail.com>).

> Icons were based on the [Font-Awesome](http://fontawesome.io/icons/) icon set.

Reviewer: It would be great if you can help update it to https:
```suggestion
Icons were based on the [Font-Awesome](https://fontawesome.com/icons) icon set.
```

> ## Redesign

Reviewer: Add an empty line:
```suggestion

## Redesign
```

Redesigned icons has been created by Nowshed H. Imran (<nowshed.imran@gmail.com>). There are some additional icons available in [flow-icon](https://github.com/nowshed-imran/flow-icon) repository that can be used in future development.
New icons are based on [la-capitaine-icon-theme](https://github.com/keeferrourke/la-capitaine-icon-theme) and [ionicons](https://github.com/ionic-team/ionicons).
`la-capitaine` project is owned by Keefer Rourke and due permission was received before modifying them.
`ionicons` project is under MIT license and thus it can be used freely.
Both of these projects can be used as reference for future icon designs.

> ## Color Reference

Reviewer: ```suggestion

## Color Reference
```

# qBittorrent Icons

## Previous
Old icons were created by Bert Verhelst (<verhelstbert@gmail.com>).
Icons were based on the [Font-Awesome](https://fontawesome.com/icons) icon set.

## Redesign
Redesigned icons has been created by Nowshed H. Imran (<nowshed.imran@gmail.com>). There are some additional icons available in [flow-icon](https://github.com/nowshed-imran/flow-icon) repository that can be used in future development.
New icons are based on [la-capitaine-icon-theme](https://github.com/keeferrourke/la-capitaine-icon-theme) and [ionicons](https://github.com/ionic-team/ionicons).
`la-capitaine` project is owned by Keefer Rourke and due permission was received before modifying them.
`ionicons` project is under MIT license and thus it can be used freely.
Both of these projects can be used as reference for future icon designs.

## Color Reference

> Colors have been selected from QT documented [SVG Colors](https://doc.qt.io/qt-6/qcolorconstants.html#svg-colors). It is recommended to use color Hex from the given source.

Reviewer 2: @xavier2k6 Should "QT" be written as "Qt" or not?

Reviewer 3: Yes, it should be Qt

# qBittorrent Icons
------------------------------------------

## Previous
Old icons were created by Bert Verhelst (<verhelstbert@gmail.com>).
Icons were based on the [Font-Awesome](https://fontawesome.com/icons) icon set.

## Redesign
Redesigned icons has been created by Nowshed H. Imran (<nowshed.imran@gmail.com>). There are some additional icons available in [flow-icon](https://github.com/nowshed-imran/flow-icon) repository that can be used in future development.
New icons are based on [la-capitaine-icon-theme](https://github.com/keeferrourke/la-capitaine-icon-theme) and [ionicons](https://github.com/ionic-team/ionicons).
`la-capitaine` project is owned by Keefer Rourke and due permission was received before modifying them.
`ionicons` project is under MIT license and thus it can be used freely.
Both of these projects can be used as reference for future icon designs.

## Color Reference

> Colors have been selected from QT documented [SVG Colors](https://doc.qt.io/qt-6/qcolorconstants.html#svg-colors). It is recommended to use color Hex from the given source.

Author: I will fix that.

# qBittorrent Icons
------------------------------------------

## Previous
Old icons were created by Bert Verhelst (<verhelstbert@gmail.com>).
Icons were based on the [Font-Awesome](http://fontawesome.io/icons/) icon set.
## Redesign

> Redesigned icons are created by Nowshed H. Imran (<nowshed.imran@gmail.com>). There are some additional icons availble in [flow-icon](https://github.com/nowshed-imran/flow-icon) repository that can be used in future development.

Reviewer 2: Typo: "availble", should be "available".
